lib/ansible/cli/config.py

```diff
@@ -35,6 +35,9 @@
 display = Display()
 
 
+_IGNORE_CHANGED = frozenset({'_terms', '_input'})
```

Comment:
Is this related to galaxy servers or just a change to how `ansible-config` displays these options for filter/lookup plugins?

Reply:
opportunistic change, it is mostly for lookup/filter/test plugins that have 'stanard documentaiton' of their inputs